back-office/multistoregeneralspecs.md

```diff
@@ -0,0 +1,185 @@
+
+**Name of the selected context**
+
+The name of the selected context is displayed, it can be:
+- All shops
+- A shop group name
+- A shop name
+
+**View my shop link**
+
+When a shop is selected, a link "view my shop" is displayed and opens the front-office of the shop.
+
+**Context drop-down**
+
+In the multistore header, there is a context drop-down allowing to select the context.
+
+In the drop-down there is a search bar with auto-completion allowing to search and select a shop or a group quickly.<br/>
+There is a placeholder: "Search shop name"<br/>
+As soon as a user types a character in the field, a search is immediately performed and returned.<br/>
+The results are displayed in a frame below the search field. <br/>
+The displayed results can then be browsed, either by the arrow keys on the keyboard, or by the mouse cursor.<br/>
+If the user chooses one of the listed results, it is automatically written in the search field and selected as context.
+
+The options listed in this drop-down are:
+- All shops with a link to select it as context
+- The shop groups names with a link to select the group as context
+- The shop names with a link to select the shop as context
+
+The colors of each shop and group are displayed before each option.<br/>
+When the user hovers the color, a tooltip is displayed "edit color"<br/>
+The user can click on the color to change it<br/>
+It opens the store's or group's page edition.
+
+If the shop URL is configured, a "view my shop" link is displayed after each shop name.<br/>
+It opens the front-office of the corresponding shop.
+
+If the shop URL isn't configured, a "configure shop URL" link is displayed after each shop name.<br/>
+It opens the store's page edition.
+
+
+### Shop association block
+
+The shop association block is displayed when you add or edit something.<br/>
+The shop association block displays all groups and all shops.<br/>
+Checkboxes allow to select / unselect groups and shops.<br/>
+It's possible to expand / collapse the wall tree and expand / collapse each group<br/>
+If you select / unselect a group, all shops of this group are selected / unselected
+
+The shop association block allows to choose for which store(s) you want to associate or disassocate the current element.
+
+When you add something, the shop(s) selected in the context are automatically checked in the shop association block.<br/>
+But it's possible to select / uncelect shops, no matter the context selected.
+
+When you edit something, you can unselect shop(s) previously selected and therefore disassocate it from these shops.<br/>
+And you can also, select shop(s) not previously selected and therefore associate it with these shops.<br/>
+If you unselect all the checkboxes (all the shops) and save, then a warning message is displayed: "Your changes will not be applied, if you want to delete this element for all your stores, you should do it from the listing" 
```

Comment:
@LouiseBonnard I just discovered a new behavior with the association block:
If you try to unselect all the shops and save then no changes are applied. I think we should advise the user of this behavior.
WDYT about the wording ?

Reply:
Hey @marionf, still need a hand on this warning message? It is planned for 1.7.8?

Reply:
@LouiseBonnard Yes, it's planned for 1.7.8